pkg/watcher/convert/convert_test.go

```diff
@@ -320,7 +320,7 @@ func toJSON(v any) []byte {
 }
 
 func TestToLogProto(t *testing.T) {
-	wantType := "results.tekton.dev/v1alpha2.Log"
+	wantType := "results.tekton.dev/v1alpha3.Log"
```

Comment:
will convert.go ever encounter v1alpha2?

Reply:
Watcher doesn't need v2. Only API results can encounter v2.

Reply:
ack thanks for confirmation @khrm